spec/markd_spec.cr

```diff
@@ -11,6 +11,11 @@ describe_spec("fixtures/regression.txt")
 
 describe_spec("fixtures/emoji.txt")
 
+# NOTE(margret): Temporarily copying strikethrough specs
+# describe_spec("fixtures/gfm-spec.txt", gfm: true)
+
+describe_spec("fixtures/gfm-extra.txt", gfm: true)
```

Comment:
I appreciate your strategic approach here, thank you.

My only suggestion is that this file be named something other than "gfm-extra". How about "working-gfm-spec.txt" or something similar?

Reply:
That would work for me. I also wanted a file to add GFM specs for stuff that wasn't covered by the normal spec (like leaving `~` if there's only one set around a char). I can split these into two different files as well.

Reply:
Actually, I think it'd be better to leave gfm-spec.txt as-is and instead of bringing pieces over to a new file piecemeal, to mark things in gfm-spec that don't/shouldn't pass as pending. Thoughts?

Reply:
Went with the latter as there are currently only ~36 failing tests in gfm-spec.txt